tools/report-converter/codechecker_report_converter/cli.py

```diff
@@ -151,8 +185,17 @@ def main():
     if 'verbose' in args and args.verbose:
         LOG.setLevel(logging.DEBUG)
 
+    metadata = args.meta if 'meta' in args else None
```

Comment:
If you remove `default=argparse.SUPPRESS` from this option then you don't need this extra line. I think we should tend to use the proper default settings through argparse module instead of applying additional logic in the argument transformation. This would be nice in CodeChecker commands too.